sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1727,7 +1727,7 @@ private Request CreateDocumentInputRequest(IEnumerable<TextDocumentInput> inputs
             request.Headers.Add(HttpHeader.Common.JsonContentType);
             request.Content = RequestContent.Create(content);
 
-            request.Headers.Add("Ocp-Apim-Subscription-Key", _subscriptionKey);
+            request.Headers.Add("Ocp-Apim-Subscription-Key", _textAnalyticsSubscriptionKey.SubscriptionKey);
```

Comment:
Let's only add this header if we're using subscription key auth ... it's not relevant to AAD.  (Same on line 1747 below)

Reply:
Issue https://github.com/Azure/azure-sdk-for-net/issues/9558 created for that